content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -42,19 +42,22 @@ for your specific needs. The following built-in configuration profiles are curre
 1. **preview**: the preview profile contains features that are experimental. This is intended to explore new features
                 coming to Istio. Stability, security, and performance are not guaranteed - use at your own risk.
 
+1. **openshift**: the openshift profile is provided as a starting point of installing Istio on an OpenShift platform.
+
 {{< tip >}}
 Some additional vendor-specific configuration profiles are also available.
 For more information, refer to the [setup instructions](/docs/setup/platform-setup) for your platform.
 {{< /tip >}}
 
 The components marked as **X** are installed within each profile:
 
-|     | default | demo | minimal | remote | empty | preview |
-| --- | --- | --- | --- | --- | --- | --- |
+|     | default | demo | minimal | remote | empty | preview | openshift |
```

Comment:
If you want to describe what components are included in the openshift profile, it should be added in the OpenShift platform setup doc, not here.